src/components/MultiSelectComponent.tsx

```diff
@@ -120,7 +120,9 @@ export class MultiSelectComponent extends React.Component<InlineComponentProps,
             let fieldList = []
             fieldList = eventKey.map((event: any) => event.value)
             this.props.modelChanged(field, fieldList)
-        } else {
+        } else if (this.props.isMulti) {
+            this.props.modelChanged(field, [])
+        } else  {
             this.props.modelChanged(field, eventKey.value)
```

Comment:
Causing error in multiselect. When we try to remove, final value in multi-select. With older version of react-select. It is returning empty list in eventKey. Now we have updated react-select to new version. It is giving null, if the eventKey is empty